docs/sources/reference/components/otelcol/otelcol.receiver.kafka.md

```diff
@@ -42,8 +42,8 @@ Name | Type | Description | Default | Required
 ---- | ---- | ----------- | ------- | --------
 `brokers` | `array(string)` | Kafka brokers to connect to. | | yes
 `protocol_version` | `string` | Kafka protocol version to use. | | yes
-`topic` | `string` | Kafka topic to read from. | _See below_ | no
-`encoding` | `string` | Encoding of payload read from Kafka. | `"otlp_proto"` | no
+`topic` | `string` | (Deprecated) Kafka topic to read from. | `""` | no
+`encoding` | `string` | (Deprecated) Encoding of payload read from Kafka. | `""` | no
```

Comment:
Deprecated in favor of what?

Reply:
I added a deprecation note under the table to say it's deprecated in favour of the `logs`, `metrics`, and `traces` arguments.